xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -322,9 +378,14 @@ viewers.Comments = Class.create({
    */
   doPreview : function(content, form) {
     form.previewButton._x_modePreview = true;
-    form.previewContent.update(content);
-    form.previewContent.show();
-    form.commentElt.hide();
+    const commentElt = jQuery(form).find(".wysiwyg-field");
```

Comment:
sounds like you're using jquery a lot here too: would be good to encapsulate it in a require(['jquery'] no?